packages/components/src/number-control/stories/index.story.tsx

```diff
@@ -23,7 +23,7 @@ const meta: Meta< typeof NumberControl > = {
 		step: { control: { type: 'text' } },
 		suffix: { control: { type: 'text' } },
 		type: { control: { type: 'text' } },
-		value: { control: null },
+		value: { control: { type: 'text' } },
```

Comment:
Note: I thought this one made more sense than what we had before.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/67581/commits/7381add898d6f77ae505ed7e5b9556b6c5375df3.

Shipping when green.